troposphere/static/js/components/projects/common/ResourceDetail.jsx

```diff
@@ -5,18 +5,33 @@ export default React.createClass({
 
     propTypes: {
         label: React.PropTypes.string.isRequired,
-        children: React.PropTypes.node.isRequired
+        children: React.PropTypes.node.isRequired,
+        onClick: React.PropTypes.func
     },
 
     render: function() {
+        let detailValue;
```

Comment:
Small point but in the spirit of Connor's formatting efforts, we are encouraged to inline attributes and destructure properties from objects for readability. 
```
const { children, onClick } = this.props;

return ( 
  <div 
     className="detail-value" 
     onClick={ onClick }
  >
     { children }
   </div>
```